### ISSUE TITLE
Rename d.service into d.service_instance 

### ISSUE THREAD

**Issue by gberche-orange** [mevansam/terraform-provider-cf#81]
Rename d.service into d.service_instance  as to  be consistent with r.service_instance

**Comment by gberche-orange** [comment]
to be handled as part of #44

**Comment by mevansam** [comment]
@gberche-orange I don't think this rename works as `d.service` returns a marketplace service definition whereas `r.service_instance` creates an instance of a `d.service`. Maybe we rename it to `d.marketplace_service` and create a new data resource `d.service_instance`, which looks up an instance of a marketplace service.
